app/config/collections.php

```diff
@@ -990,8 +991,30 @@
                 'required' => false,
                 'default' => null,
                 'array' => false,
+                'filters' => ['encrypt'],
```

Comment:
Is this useful addition? Does it serve any purpose, or just make things slower?

Reply:
It is useful, yet. If people upload MD5 hashed password, we should at least encrypt it to protect it as much as possible.

Regarding MD5, there is also the logic to re-hash the password as soon as the user logs in. That makes the timeframe when we have MD5 passwords as small as possible.